app/content/survey-manual.md

```diff
@@ -0,0 +1,261 @@
+    <pre>
+    <code class="hljs language-json">
+{
+  "publicationId": "asdf-1234",
+  "name": "Daily Bugle"
+}
+    </code>
+    </pre>    
+  </li> -->
+  <li class="flexible-list-item">
+    <img src="/img/survey.png">
+    <pre>
+    <code class="hljs language-json">
+{
+  "name": "Basic survey",
+  "configurationId": "49c12712-9750-4571-8c67-96722561c13a"
+}
+    </code>
+    </pre> 
+  </li>
+    <li class="flexible-list-item">
+    <img src="/img/survey-multi-page.png">
+    <pre>
+    <code class="hljs language-json">
+{
+  "name": "Multi-page survey",
+  "configurationId": "e98a2efb-d009-43c9-99ef-dda11c8c5a7f"
+}
+    </code>
+    </pre> 
+  </li>
+</ul>
+
+After creating a configuration, the Publisher Center page will provide a `configurationId` in response for each
+newsletter configuration. These ids will then be used in subsequent javascript api calls.
+
+```json
+[
+  {
+    name: 'Basic survey',
+    configurationId: '49c12712-9750-4571-8c67-96722561c13a',
+  },
+  {
+    name: 'Multi-page survey',
+    configurationId: 'e98a2efb-d009-43c9-99ef-dda11c8c5a7f',
+  },
+]
+```
+
+### Invoke Survey Prompts
+
+To configure survey prompts, `swg.js` must first be configured on the page.
+These examples show using the initialization of the library in `manual` mode, but the
+APIs are also available in automatic mode. 
+
+### Get the prompt instance to display
+
+To invoke a survey prompt, a publisher must use the `configurationId` provided by
+Google in response to submitting a prompt configuration. Publishers use the
+`configurationId` to fetch a valid prompt instance using the `subscriptions.getAvailableInterventions()` method from the initialized `swg.js` library.
+
+```javascript
+const publisherConfiguration = {
+  name: 'Basic survey',
+  configurationId: '49c12712-9750-4571-8c67-96722561c13a',
+};
+
+const availableInterventions = await subscriptions.getAvailableInterventions();
+
+const prompt = availableInterventions.find(({configurationId}) => {
+    return configurationId === publisherConfiguration.configurationId;
+});
+```
+
+### Show the prompt
+
+To display a prompt, use the returned value from `subscriptions.getAvailableInterventions()` and use the `show` method:
+
+```javascript
+prompt?.show({
+    isClosable: true,
+    onResult: (result) => {
+        //Store the result, which is the email of the newsletter signup.
+
+        //Return true to let Google know that you have received and processed
+        //the returned email.
+        return true;
+    }
+});
+```
+
+### Handle the response
+
+The `onResult` callback will include information on the configuration used
+to create the prompt, as well as the newsletter subscriber's information. The `configurationId` matches the `configurationId provided to the publisher from Google, in response to the configuration authored by the publisher in the initial step.
+
+!!! note `onResult()` callback format is variable.
```

Comment:
The type is documented here, and I think we're gonna try pretty hard to not change it https://github.com/subscriptions-project/swg-js/blob/main/src/api/available-intervention.ts:InterventionResult

Reply:
ops, yes!